src/verify_cert.rs

```diff
@@ -723,31 +724,36 @@ mod tests {
         trust_anchor: ChainTrustAnchor,
     ) -> ControlFlow<Error, Error> {
         let ca_cert = make_issuer("Bogus Subject");
-        let ca_cert_der = CertificateDer::from(ca_cert.serialize_der().unwrap());
+        let ca_cert_der = ca_cert.cert.der().clone();
```

Comment:
`ca_cert.cert.into()`? Or do we need to keep `ca_cert` intact here? (It looks like it might be okay.)

Reply:
>  we need to keep ca_cert intact here?

Yup, it's used as an issuer for the first iteration of the loop starting on L736.